test/API/driver/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 2.8.12.2 FATAL_ERROR)
```

Comment:
minimum is 3.18 now

Reply:
Right but seeing as this is Kitware's system library, our minimum doesn't exactly apply across everything. Though as noted below FetchContent could be used rather than having the driver in place. It was a topic of discussion that still needed to take place because it was unclear how to handle the test driver code.